create_aio_app/template/{{cookiecutter.project_name}}/conftest.py

```diff
@@ -1,14 +1,14 @@
 import pytest
-{%- if cookiecutter.use_postgres == 'y' %}
+{%- if cookiecutter.use_postgres == 'aiopg' %}
 from sqlalchemy import create_engine
 import aiopg.sa
 {%- endif %}
 
-from {{ cookiecutter.project_name }}.utils.common import PATH, get_config
-from {{ cookiecutter.project_name }}.app import init_app
-{%- if cookiecutter.use_postgres == 'y' %}
-from {{ cookiecutter.project_name }}.migrations import metadata
-from {{ cookiecutter.project_name }}.users.tables import users
+from {{cookiecutter.project_name}}.utils.common import PATH, get_config
```

Comment:
In the whole project uses `{{ cookiecutter.project_name }}` with indent, so don't change that, please.
In [jinja2 documentation](http://jinja.pocoo.org/docs/2.10/) also use indents

```
{% for user in users %}
  <li><a href="{{ user.url }}">{{ user.username }}</a></li>
{% endfor %}
```

Reply:
sorry autoformatting, my bad.